lrlex/src/lib/ctbuilder.rs

```diff
@@ -78,11 +78,47 @@ pub enum RustEdition {
     Rust2021,
 }
 
+use proc_macro2::TokenStream;
+use quote::{ToTokens, TokenStreamExt};
+/* Quote by default quotes option values as an empty string for `None`,
```

Comment:
I think we can use `//` here :)

Reply:
I just rewrote all this as docstrings, and adding one for all those wrapper types.
So should be fixed in [f6d33f4](https://github.com/softdevteam/grmtools/pull/494/commits/f6d33f49b5d685aa9952122f6640f80433926fdc)